packages/forms-web-app/src/routes/file-based-router/comment-planning-appeal/index.js

```diff
@@ -1,13 +0,0 @@
-const express = require('express');
```

Comment:
Note to our future selves: We can reinstitute this explicit 404 once https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3730 is merged

Reply:
I'll add a comment to this effect